src/index.js

```diff
@@ -3,9 +3,10 @@ var WriteData = require('./write-data');
 
 var main = function(options) {
   console.log("Gathering Manifest data...");
-  var resources = WalkManifest(options.decrypt, options.output, options.input);
-
-  return WriteData(options.decrypt, options.concurrency, resources);
+  var resources = [];
+  WalkManifest(options.decrypt, options.output, options.input, false, 0, resources, function(){
```

Comment:
why pass resources into WalkManifest? I would expect the callback to be a nodeback with an error and resources params.